applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -337,19 +337,42 @@ static const struct bt_bap_unicast_server_cb unicast_server_cb = {
 static void stream_recv_cb(struct bt_bap_stream *stream, const struct bt_iso_recv_info *info,
```

Comment:
The `stream_recv_cb` function for multiple roles seems to be identical. Did you consider a common callback function or some other common function to handle this without duplicating code?

Reply:
Did consider it, decided it should be a separate PR, but then went ahead and fixed it now :P